worker.d.ts

```diff
@@ -0,0 +1,61 @@
+import { Action, Store } from "unistore";
+
+/**
+ * Actions object.
+ */
+export interface Actions<WorkerState> {
+	[action: string]: Action<WorkerState>;
+}
+
+/**
+ * Types of actions received by registerActions.
+ */
+export type ActionRegister<WorkerState> = (
+	store: WorkerStore<WorkerState>
+) => Actions<WorkerState> | Actions<WorkerState>;
+
+/**
+ * Enhanced unistore store.
+ */
+export interface WorkerStore<WorkerState> extends Store<WorkerState> {
+
+	/**
+	 * Creates a new instance of WorkerStore.
+	 */
+	new(): WorkerStore<WorkerState>;
+
+	/**
+	 * Retrieves the given action.
+	 * @param action Action to retrieve (Worker actions take strings).
+	 */
+	action(action: Action<WorkerState> | string): (...params: any[]) => void;
```

Comment:
I think this one is already defined on `Store`, right?

Reply:
https://github.com/developit/unistore/blob/master/index.d.ts#L12

Reply:
Hi @developit, no worries about the time to review (with the recent release of Preact X, I understand). About your question, Unistore's action doesn't work exactly like Stockroom's action. Stockroom takes an `Action` or a `string`, and Unistore (afaik) only takes an `Action`. Also Stockroom returns `void` where Unistore returns the `BoundAction`.